nemo/collections/tts/torch/helpers.py

```diff
@@ -16,14 +16,15 @@
 
 import numpy as np
 import torch
+from einops import rearrange
 from scipy import ndimage
-from scipy.stats import betabinom
+from torch.special import gammaln
 
 
 class BetaBinomialInterpolator:
```

Comment:
I think there are two small changes missing in the BetaBinomialInterpolator. 

1. The default border setting for the zoom function is to use constant padding, which can result in a column of zero probabilities. I think this is easily resolved using mode='nearest'.
2. The probabilities for phoneme_token given mel frame do not exactly add up to 1 after interpolation (just slightly off), so a renormalization would be necessary.

Reply:
Hi! I agree with both points, that would be a more correct implementation.

However, I'd rather deprecate the whole `BetaBinomialInterpolator`. It's is a cache that is not needed anymore precisely due to changes in this PR.

Reply:
Right! I totally missed that part :)